plugins/stats/stats.go

```diff
@@ -170,9 +170,10 @@ func (p StatsPlugin) sendEventStats(id string, stats *dockerclient.Stats, ec cha
 	cName := cInfo.Name[1:]
 	cNamePath := fmt.Sprintf(cName)
 
-	if cInfo.Node.ID != "" {
-		cNamePath = fmt.Sprintf("nodes.%s.%s", cInfo.Node.Name, cName)
-	}
+	/*
```

Comment:
I was getting a `cInfo.Node undefined (type *dockerclient.ContainerInfo has no field or method Node)`. I'll just take it out. I think go-plus wasn't noticing the vendor directory I assume.